processor/traceidprocessor/README.md

```diff
@@ -0,0 +1,32 @@
+# TraceID processor
+
+Replaces the high order 32 bits of the `TraceID` to contain the epoch value in seconds based on the start time of the span if available, otherwise takes the current Unix epoch time. The computed epoch value of the first span with a `TraceID` is cached for subsequent spans with the same `TraceID` to make sure they end up in the same trace.
+
+The following settings are required:
+
+- `cache_endpoint`: contains the endpoint of the cache implementation. Cache provider uses the URL scheme to switch between local map imeplementation or Redis cache. For local map use `local://` as the endpoint (**_For local testing only. Not for production use._**). For Redis cache use `redis://<host>:<port>`.
+- `ttl` : contains the time to live in seconds for the computed epoch value. Only used for Redis provider.
+
+Example:
+
+```yaml
+receivers:
+  zipkin:
```

Comment:
Since zipkin instrumentation uses x-ray compatible IDs, it seems like a bad example. Jaeger? Out of curiosity what instrumentation does the user you're trying to help use? For OpenTelemetry users, we really want them to use x-ray ID generation to avoid this overhead / complexity.

Reply:
This is based on behaviour observed on Istio service mesh. The zipkin trace exporter packaged with Istio generates 128 bit random UID which breaks with X-Ray.